File: go/052624_B4_17010.go
/*
 * https://www.acmicpc.net/problem/17010
 */
package main

import (
	"fmt"
	"os"
	"bufio"
	"strings"
)

func main() {
	rw := bufio.NewReadWriter(bufio.NewReader(os.Stdin), bufio.NewWriter(os.Stdout))
	defer rw.Flush()

	var L int
	fmt.Fscanf(rw, "%d\n", &L)
	for i := 0; i < L; i++ {
		var N int
		var x rune
		fmt.Fscanf(rw, "%d %c\n", &N, &x)
		fmt.Fprintf(rw, "%s\n", strings.Repeat(string(x), N))
	}
}

File: python/052624_B4_17010.py
# https://www.acmicpc.net/problem/17010
import sys

L = int(sys.stdin.readline())
answer = []
for i in range(L):
    N, x = sys.stdin.readline().split()
    answer.append(x*int(N)+"\n")
sys.stdout.write(''.join(answer))
